stock-tracker/src/loading.css

```diff
@@ -0,0 +1,33 @@
+.lds-ripple {
+  display: flex;
+  margin: 10% auto;
+  position: relative;
+  width: 64px;
+  height: 64px;
+}
+.lds-ripple div {
+  position: absolute;
+  border: 4px solid #fff;
+  opacity: 1;
+  border-radius: 50%;
+  animation: lds-ripple 1s cubic-bezier(0, 0.2, 0.8, 1) infinite;
+}
+.lds-ripple div:nth-child(2) {
+  animation-delay: -0.5s;
+}
+@keyframes lds-ripple {
+  0% {
+    top: 20px;
+    left: 20px;
+    width: 0;
+    height: 0;
+    opacity: 1;
+  }
+  100% {
+    top: -1px;
+    left: -1px;
+    width: 40px;
+    height: 40px;
+    opacity: 0;
+  }
+}
```

Comment:
nit: I recommend leaving an empty line at the end of a file. It reduces the "change count" detected by Git, resulting in better highlighting of what has _actually_ changed.